generators/server/templates/build.gradle.ejs

```diff
@@ -443,6 +443,11 @@ if (SPRING_BOOT_VERSION.indexOf('M') > -1 || SPRING_BOOT_VERSION.indexOf('RC') >
     implementation "org.springframework.cloud:spring-cloud-starter-stream-kafka"
     testImplementation "org.testcontainers:kafka"
 <%_ } _%>
+<%_ if (messageBrokerPulsar) { _%>
+    implementation "org.springframework.cloud:spring-cloud-stream"
+    implementation "org.springframework.pulsar:spring-pulsar-spring-cloud-stream-binder:${springPulsarVersion}"
+    testImplementation "org.testcontainers:pulsar"
+<%_ } _%>
```

Comment:
Add to it's own pulsar.gradle like [cache.gradle](https://github.com/jhipster/generator-jhipster/blob/main/generators/spring-cache/templates/gradle/cache.gradle.ejs)
And inject like
https://github.com/jhipster/generator-jhipster/blob/f448a5a3fbe27967df92495080b2391140132648/generators/spring-cache/generator.mts#L62-L66

Reply:
We don't do that for other deps (kafka, swagger, ...).
Shouldn't this be done in another PR and keep consistency for the moment ?

Reply:
I've started to split recently when working on a specific generator.
Otherwise the PR can grow too much.
Kafka is done at https://github.com/jhipster/generator-jhipster/pull/21577.

Reply:
OK. I'll do it then.